pkg/config/types/performance.go

```diff
@@ -0,0 +1,68 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Performance = string
+
+const (
```

Comment:
As above, let's do "none", "mutagen", and "nfs"

Reply:
Done